mangaki/mangaki/admin.py

```diff
@@ -76,6 +191,10 @@ class WorkAdmin(admin.ModelAdmin):
         'controversy',
     )
 
+    def get_queryset(self, request):
+        qs = super().get_queryset(request)
+        return qs.filter(redirect__isnull=True)  # Exclude phantom works
```

Comment:
Done: `# Equivalent to default_manager_name = 'all_objects', because first in the list`.